terraform-module/modules/frontend-spa-cdn/main.tf

```
@@ -148,6 +142,36 @@ resource "aws_cloudfront_distribution" "this" {
   }
 }
 
+resource "aws_cloudwatch_log_delivery_source" "cloudfront_log_source" {
+  region = "eu-west-1"
+
+  name         = "cloudfront_log_source"
+  log_type     = "ACCESS_LOGS"
+  resource_arn = aws_cloudfront_distribution.this.arn
+}
+
+resource "aws_cloudwatch_log_delivery_destination" "cloudfront_log_destination" {
+  region = "eu-west-1"
+
+  name          = "cloudfront_log_destination"
+  output_format = "parquet"
+
+  delivery_destination_configuration {
+    destination_resource_arn = module.data_aws_core.s3_bucket_log.bucket_domain_name
+  }
+}
+
+resource "aws_cloudwatch_log_delivery" "cloudfront_log_delivery" {
+  region = "eu-west-1"
+
+  delivery_source_name     = aws_cloudwatch_log_delivery_source.cloudfront_log_source.name
+  delivery_destination_arn = aws_cloudwatch_log_delivery_destination.cloudfront_log_destination.arn
+
+  s3_delivery_configuration {
+    suffix_path = "/CloudFront/{DistributionId}/{yyyy}/{MM}/{dd}/{HH}"
```

let's use lowercase everywhere

```suggestion
    suffix_path = "/cloudfront/{DistributionId}/{yyyy}/{MM}/{dd}/{HH}"
```